lib/metrics.js

```diff
@@ -120,7 +120,7 @@ module.exports = {
 
       if (eventName.startsWith('core:') || eventName.startsWith('editor:')) return
 
-      if (!(eventName.indexOf(':') > -1)) return
+      if (!eventName.includes(':')) return
```

Comment:
:heart_eyes:

Reply:
@thomasjo: Thanks for [pointing out](https://github.com/atom/metrics/pull/88#issuecomment-396555060) this opportunity for improvement! :zap: